import-areas/import-areas.go

```diff
@@ -160,11 +160,24 @@ func postCountryQuery(query string) (*models.Area, error) {
 	v.Set("query", query)
 
 	client := &http.Client{}
-	req, _ := http.NewRequest("POST", "http://statistics.data.gov.uk/sparql", strings.NewReader(v.Encode()))
+	req, err := http.NewRequest("POST", "http://statistics.data.gov.uk/sparql", strings.NewReader(v.Encode()))
+	if err != nil {
+		log.Event(ctx, "error returned from building the country request", log.ERROR, log.Error(err))
+		return nil, err
+	}
+
 	req.Header.Add("Accept", "application/sparql-results+json")
-	resp, _ := client.Do(req)
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Event(ctx, "error returned from country POST request", log.ERROR, log.Error(err))
+		return nil, err
+	}
```

Comment:
Add defer close immediately after this block & check the response status before you read/unmarshal the body.

Reply:
I have also updated this

Reply:
See comment below about defer close.